h/presenters/group_json.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+
+class GroupJSONPresenter(object):
+    """Present a group in the JSON format returned by API requests."""
+
+    def __init__(self, group, route_url=None):
+        self.group = group
+        self._route_url = route_url
```

Comment:
It bugs me that I need `request.route_url` in here, but I can't think of a better way to handle it. Sticking yet another service layer in here somewhere seems overdesigned. At present I've made it so the absence of `route_url` won't bring things to a grinding halt.